components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -487,4 +487,24 @@ - (void)testMessageStaysWhenFocusOnShowIsEnabled {
   XCTAssertFalse(self.manager.internalManager.currentSnackbar.accessibilityElementsHidden);
 }
 
+- (void)testMessageCustomizationUsingWillPresentBlock {
+  // Given
+  self.message.snackbarMessageWillPresentBlock =
+      ^(MDCSnackbarMessage *snackbarMessage, MDCSnackbarMessageView *messageView) {
+        messageView.backgroundColor = UIColor.blueColor;
+      };
+
+  // When
+  [self.manager showMessage:self.message];
+  XCTestExpectation *expectation = [self expectationWithDescription:@"completed"];
+  dispatch_async(dispatch_get_main_queue(), ^{
+    [expectation fulfill];
+  });
+  [self waitForExpectationsWithTimeout:3 handler:nil];
+
+  // Then
+  XCTAssertEqualObjects(self.manager.internalManager.currentSnackbar.backgroundColor,
```

Comment:
Nit: instead of using a `UIColor`, it may be cleaner to have a `BOOL` (`isBlockCalled` or something) that is set to true by the block invocation.

Reply:
done